### ISSUE TITLE
Нет информации о причине, по которой не найдено загруженных тестов.

### ISSUE THREAD

**Issue by BarinovIN** [silverbulleters/add#1095]
**Описание ошибки**
Выполнение дымовых тестов в пакетном режиме не начинается с сообщением "Не найдено загруженных тестов. Выполнение тестов завершается" и причина, по которой тесты не найдены совершенно неизвестна остаётся. Простой перезапуск, без каких-либо изменений и тесты находятся и выполняются.﻿

В журнале регистрации только эти записи:
![Журнал регистрации](https://user-images.githubusercontent.com/65981489/234505889-9df4eca6-3ec3-4694-b649-bd89d521e7d2.png)

В логе выполнения:
```
ИНФОРМАЦИЯ - Техническая информация:
ИНФОРМАЦИЯ - ...
ИНФОРМАЦИЯ - Активные расширения:
ИНФОРМАЦИЯ - Не найдено загруженных тестов. Выполнение тестов завершается
ИНФОРМАЦИЯ - Не найдено результатов тестирования
ИНФОРМАЦИЯ - Выполнение команды/действия в режиме 1С:Предприятие завершено.
ПРЕДУПРЕЖДЕНИЕ - 
Vanessa-ADD или 1С:Предприятие вернуло код возврата 1
ОШИБКА - Часть тестов упала!
```

**Сценарий воспроизведения**
1. Запустить тесты в пакетном режиме
2. Иногда, при неизвестных условиях, тесты не запускаются, выдаётся сообщение: "Не найдено загруженных тестов. Выполнение тестов завершается"

**Ожидаемое поведение, если бы ошибки не было**
Перед сообщением "Не найдено загруженных тестов. Выполнение тестов завершается" есть сообщение о том, что пошло не так. Описание ошибки из исключения или другое указание на причину. В журнале регистрации подробное описание ошибки, если возникло исключение.

**Окружение:**
   ```
 Техническая информация:
	- Версия Vanessa-ADD: 6.9.4
	- Имя конфигурации: СовсемНеТиповая
	- Платформа 1С: 8.3.19.1351
	- Тип базы: Клиент-серверная ИБ
	- Режим запуска: Управляемое приложение (Тонкий клиент)
	- Режим совместимости (для TestManager): Версия8_3_14
	- Режим совместимости интерфейса (для TestManager): ТаксиРазрешитьВерсия8_2
	- Режим синхронности: включен
	- Защита от опасных действий: выключен
	- Язык (для TestManager): Русский
	- Локализация (для TestManager): ru_RU
	- Операционная система (клиент): Windows x86-64 Microsoft Windows 10 version 10.0  (Build 19044)
	- Операционная система (сервер): Windows x86-64 Microsoft Windows 8.1 version 6.3  (Build 9600)
	- Активные расширения:
```

**Comment by artbear** [comment]
@BarinovIN как правило, подобное сообщение "Не найдено загруженных тестов. Выполнение тестов завершается" говорит о том, что 
- или ошиблись с путем к тестам
- или файл с тестами имеет синтаксическую ошибку и не открывается в режиме Предприятия

ПР смотрю, думаю

**Comment by BarinovIN** [comment]
Да, но в этих случаях, простой перезапуск тестов без изменения настроек будет стабильно давать тот же результат. А у меня картина иная. После обработки исключения причина стала видна: #1097 . Но пока не могу сказать, что понятна.

> файл с тестами имеет синтаксическую ошибку и не открывается в режиме Предприятия

В этом случае удобно сразу знать в каком файле проблема. Теперь это будет видно.
